soco/core.py

```diff
@@ -1490,6 +1493,59 @@ def get_item_album_art_uri(self, item):
         else:
             return None
 
+    @only_on_master
+    def set_sleep_timer(self, sleep_time_seconds):
+        """Sets the sleep timer.
+
+        Args:
+            sleep_time_seconds (`int`): How long to wait before turning off
+                speaker in seconds. Maximum value of 86399
+
+        Returns:
```

Comment:
As mentioned in the conversation. We would rather move to not doing this. Failure should be indicated by an exception and it is a success otherwise.